Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.controller.ts

```diff
@@ -52,24 +51,24 @@
                 var fromDate = moment(concluded, [Kitos.Constants.DateFormat.DanishDateFormat, formatDateString]).startOf("day");
                 var endDate = moment(expirationDate, [Kitos.Constants.DateFormat.DanishDateFormat, formatDateString]).endOf("day");
                 var date = moment(value, Kitos.Constants.DateFormat.DanishDateFormat);
-
+                
                 if (value === "" || value == undefined) {
                     var payload = {};
                     payload[field] = null;
-                    patch(payload, $scope.autoSaveUrl + "?organizationId=" + user.currentOrganizationId);
+                    patch(payload, saveUrlWithOrgId)
+                        .then(_ => reloadValidationStatus());
```

Comment:
you are also calling it in the pathc method that you just called.. is 1 time not enough?